File: NirvanaSMS.Api.CSharp/Core/Entity/StatusCode.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace NirvanaSMS.Api.Core.Entity
{
    public class StatusCode
    {
        public string Code { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public bool IsSuccess { get; set; }

        public StatusCode()
        {

        }

        public StatusCode(string code, string name, string description)
        {
            Code = code;
            Name = name;
            Description = description;
        }

        public StatusCode(string code, string name, string description, bool isSuccess)
        {
            Code = code;
            Name = name;
            Description = description;
            IsSuccess = isSuccess;
        }
    }
}

File: NirvanaSMS.Api.Java/src/main/java/org/nirvanasms/api/core/entity/StatusCode.java
package org.nirvanasms.api.core.entity;

public class StatusCode {
	private String code;
	private String name;
	private String description;
	private boolean isSuccess;
	
	public StatusCode(){}
	
	public StatusCode(String code, String name, String description){
		this.code = code;
		this.name = name;
		this.description = description;
	}
	
	public StatusCode(String code, String name, String description, boolean isSuccess){
		this.code = code;
		this.name = name;
		this.description = description;
		this.isSuccess = isSuccess;
	}

	public String getCode() {
		return code;
	}

	public void setCode(String code) {
		this.code = code;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getDescription() {
		return description;
	}

	public void setDescription(String description) {
		this.description = description;
	}

	public boolean getIsSuccess() {
		return isSuccess;
	}

	public void setIsSuccess(boolean isSuccess) {
		this.isSuccess = isSuccess;
	}
	
	
}
